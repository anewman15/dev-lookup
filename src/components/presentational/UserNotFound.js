import common from '../../styles/commonStyles.module.css';

const UserNotFound = () => (
  <div className={`${common.container} ${common.flexCenter}`}>
    <p className={`${common.boxRed}`}>User not found</p>
  </div>
);

export default UserNotFound;
