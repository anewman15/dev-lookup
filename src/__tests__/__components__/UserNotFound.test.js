import { unmountComponentAtNode } from 'react-dom';
import renderer from 'react-test-renderer';
import UserNotFound from '../../components/presentational/UserNotFound';

let container = null;
beforeEach(() => {
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

describe('UserNotFound', () => {
  it('renders the user not found message', () => {
    const tree = renderer
      .create(<UserNotFound />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
