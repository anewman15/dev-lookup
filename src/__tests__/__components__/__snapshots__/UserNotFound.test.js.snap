// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserNotFound renders the user not found message 1`] = `
<div
  className="container flexCenter"
>
  <p
    className="boxRed"
  >
    User not found
  </p>
</div>
`;
